.gitignore

```
@@ -4,3 +4,21 @@ project/
 *.jou
 *.log
 *.str
+sim/sim_wb8_old/wb8_agent_slave.sv
+sim/sim_wb8_old/wb8_agent.sv
+sim/sim_wb8_old/wb8_config.sv
+sim/sim_wb8_old/wb8_coverage.sv
+sim/sim_wb8_old/wb8_driver_slave.sv
+sim/sim_wb8_old/wb8_driver.sv
+sim/sim_wb8_old/wb8_environment.sv
+sim/sim_wb8_old/wb8_monitor_slave.sv
+sim/sim_wb8_old/wb8_monitor.sv
+sim/sim_wb8_old/wb8_pkg.sv
+sim/sim_wb8_old/wb8_scoreboard_func.sv
+sim/sim_wb8_old/wb8_scoreboard.sv
+sim/sim_wb8_old/wb8_sequence_item.sv
+sim/sim_wb8_old/wb8_sequence_slave.sv
+sim/sim_wb8_old/wb8_sequence.sv
+sim/sim_wb8_old/wb8_tb_top.sv
+sim/sim_wb8_old/wb8_test.sv
+sim/sim_wb8_old/wb8_top_interface.sv
```

If you want to gitignore the whole dir, just put a line


```suggestion
sim/sim_wb8_old
```

Better yet, just put it in a branch in your fork

---

skrg dah ada di fork old

---

Are you sure? https://github.com/AgapeDsky/i2c_master_wb-2agent/tree/old/sim/sim_wb8

If done then you can make a commit to your main branch and we can resolve